migration/src/test/java/com/intershop/customization/migration/gradle/UpdateGradleBuild7to10Test.java

```diff
@@ -1,29 +1,26 @@
 package com.intershop.customization.migration.gradle;
 
-import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
there is only one import. please use old import
```suggestion
import static org.junit.jupiter.api.Assertions.assertEquals;
```